modules/environment/nix.nix

```diff
@@ -75,6 +91,7 @@ in
         "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY="
         "nix-on-droid.cachix.org-1:56snoMJTXmDRC1Ei24CmKoUqvHJ9XCp+nidK7qkMQrU="
       ];
+      experimentalFeatures = mkIf (versionAtLeast cfg.package.version "2.4") [ "nix-command" ];
```

Comment:
https://nixos.org/manual/nix/stable/command-ref/conf-file.html says it's enabled by default in stable (2.4), do we still need to worry about it?

Reply:
or is this exactly why we have to worry about it? =)

Reply:
Fun fact: Yesterday the default was 0, I wondered what that meant.. Seemed to be a bug in the documentation :D Then we definitely do not need this, I will remove it.